server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -274,23 +274,31 @@ export class DocumentStorage implements IDocumentStorage {
 			lumberjackProperties,
 		);
 
+		const document: IDocument = {
+			createTime: Date.now(),
+			deli: JSON.stringify(deli),
+			documentId,
+			session,
+			scribe: JSON.stringify(scribe),
+			tenantId,
+			version: "0.1",
+			storageName,
+			isEphemeralContainer,
+		};
+		const documentDbValue: IDocument & { ttl?: number } = {
+			...document,
+		};
+		if (isEphemeralContainer) {
+			documentDbValue.ttl = 60 * 60 * 24; // 24 hours in seconds
```

Comment:
Two things:
1. Let's get @yangg-msft  and @pradeepvairamani  feedback on this
2. Once merged, we need to set a colllection ttl that never expire, and this document level would override that behavior to have functional

Reply:
I think this can be merged without risk, and once the TTL is added on FRS side, it will allow cleanup to happen automatically.